src/AWS.Deploy.Orchestration/CdkProjectHandler.cs

```diff
@@ -124,20 +125,47 @@ public async Task DeployCdkProject(OrchestratorSession session, CloudApplication
             _interactiveService.LogSectionStart("Deploying AWS CDK project",
                 "Use the CDK project to create or update the AWS CloudFormation stack and deploy the project to the AWS resources in the stack.");
 
-            var deploymentStartDate = DateTime.Now;
             // Handover to CDK command line tool
             // Use a CDK Context parameter to specify the settings file that has been serialized.
-            var cdkDeploy = await _commandLineWrapper.TryRunWithResult($"npx cdk deploy --require-approval never -c {Constants.CloudFormationIdentifier.SETTINGS_PATH_CDK_CONTEXT_PARAMETER}=\"{appSettingsFilePath}\"",
+            var cdkDeployTask = _commandLineWrapper.TryRunWithResult( $"npx cdk deploy --require-approval never -c {Constants.CloudFormationIdentifier.SETTINGS_PATH_CDK_CONTEXT_PARAMETER}=\"{appSettingsFilePath}\"",
                 workingDirectory: cdkProjectPath,
                 environmentVariables: environmentVariables,
                 needAwsCredentials: true,
                 redirectIO: true,
                 streamOutputToInteractiveService: true);
 
-            await CheckCdkDeploymentFailure(cloudApplication, deploymentStartDate);
+            var cancellationTokenSource = new CancellationTokenSource();
+            var retrieveStackIdTask = RetreiveStackId(cloudApplication, cancellationTokenSource.Token);
+
+            var deploymentStartDate = DateTime.Now;
```

Comment:
To enforce good habits lets use `UtcNow`.

Reply:
I reverted to Now because we are filtering the CloufFormation stack events based on the timestamp which causes a mismatch.

Reply:
The CloudFormation SDK is still returning in local time, which is why I switched back to local time.

Reply:
Thanks for moving to UTC with the `.ToUniversalTime`